tock/hours/forms.py

```diff
@@ -183,8 +194,12 @@ def clean(self):
                         ),
                     )
             else:
-                if ('project_allocation' in self.cleaned_data and \
-                   self.cleaned_data['project_allocation'] != ''):
+                if (
+                    "project_allocation" in self.cleaned_data
+                    and not self._is_empty_or_zero(
```

Comment:
do we need the `staticmethod` decorator here? Shouldn't we call it as `TimecardObjectForm._is_empty_or_zero` to be clearer?

Reply:
We don't strictly need to use `@staticmethod`, but I think it's good practice when we don't need to use any properties of the instance or the class. As far as calling the method, `self` works and is much less verbose than having to repeat the class name.